Project.toml

```diff
@@ -1,26 +1,40 @@
 name = "QEDcore"
 uuid = "35dc0263-cb5f-4c33-a114-1d7f54ab753e"
-authors = ["Uwe Hernandez Acosta <u.hernandez@hzdr.de>", "Anton Reinhard <a.reinhard@hzdr.de>"]
+authors = [
+    "Uwe Hernandez Acosta <u.hernandez@hzdr.de>",
+    "Anton Reinhard <a.reinhard@hzdr.de>",
+]
 version = "0.1.0"
```

Comment:
Since it's a push to main, should we update the version?

Reply:
I think the important property for a version is not the commit which bumps the version in the Project.toml, but the respective tag of main. Since we haven't had a tag for this version (the bug was fixed before that) the commit which merges the hotfix can be tagged as v0.1.0 (after registration). 
This is also the reason, why we should not squash this 🙆‍♂️

Reply:
Ok yeah, that sounds reasonable.